webapp/README.md

```diff
@@ -4,22 +4,19 @@
 
 ```bash
 # install dependencies
-$ npm install
+$ yarn install
 
 # serve with hot reload at localhost:3000
 $ npm run dev
 
 # build for production and launch server
 $ npm run build
 $ npm run start
-
-# generate static project
-$ npm run generate
 ```
 
 For detailed explanation on how things work, check out [Nuxt.js docs](https://nuxtjs.org).
 
 ## Deployment Target
 
-- Server
-- Da für das Backend ein Server mit Node.js benötigt wird, wird das Frontend auch als node.js Prozess deployet. Da der aufwand sowieso da ist, halten sich die Vorteile für die Static Variante in grenzen.
+- Type: Server
+- Da für das Backend ein Server mit Node.js benötigt wird, wird das Frontend auch als node.js Prozess deployet. Da der Aufwand sowieso vorhanden ist, halten sich die Vorteile für die "Static" Variante in Grenzen.
```

Comment:
⭐ For instructions in the README.md on how to build your webapp for production.
Good description! It could be described a bit more detailed, but otherwise it is good. The mix of english / german languages is also not understandable for everyone, but for me it was no problem :)

Reply:
I like that you explained your decision :+1: a reason to have the frontend statically deployed but a backend running on a NodeJS process could be that 90% of your pages might be independent of the backend. You might want the frontend to be deployed on a file hoster to benefit from the performance gains (JAMstack).